data/avro-schema/src/main/resources/avro/net/corda/data/virtualnode/VirtualNodeOperationStatusResponse.avsc

```diff
@@ -0,0 +1,18 @@
+{
+  "type": "record",
+  "name": "VirtualNodeOperationStatusResponse",
+  "namespace": "net.corda.data.virtualnode",
+  "fields": [
+    {
+      "name": "requestId",
+      "type": "string"
+    },
+    {
+      "name": "status",
```

Comment:
since this is list, might be best to name this something slightly different, just thinking out aloud:
- statuses
- statusList
- operationHistory
- requestHistory
- requestStatuses
- statusHistory
- requestHistory

Possibly requestHistory is jumping out at me the most, what do you think?

Reply:
Thanks, definitely should be something other than "status". Thinking through them, I like "operationHistory" as there will be one entry for each operation that was made, rather than for each request or individual status change. I'll go with this for now, but happy to change this if we prefer one of the others or something else altogether.